lib/contify_api/connection.ex

```diff
@@ -105,17 +105,28 @@ defmodule ContifyAPI.Connection do
     app_secret = app_secret()
     app_id = app_id()
 
-    timeout = Application.get_env(:contify, :timeout, @default_timeout)
-
-    [
-      {Tesla.Middleware.BaseUrl, base_url},
-      {Tesla.Middleware.Headers, [{"user-agent", user_agent}]},
-      {Tesla.Middleware.Headers, [{"APPSECRET", app_secret}]},
-      {Tesla.Middleware.Headers, [{"APPID", app_id}]},
-      {Tesla.Middleware.Timeout, timeout: timeout},
-      {Tesla.Middleware.EncodeJson, engine: json_engine}
-      | middleware
-    ]
+    if Mix.env() == :test do
+      [
+        {Tesla.Middleware.BaseUrl, base_url},
+        {Tesla.Middleware.Headers, [{"user-agent", user_agent}]},
+        {Tesla.Middleware.Headers, [{"APPSECRET", app_secret}]},
+        {Tesla.Middleware.Headers, [{"APPID", app_id}]},
+        {Tesla.Middleware.EncodeJson, engine: json_engine}
+        | middleware
+      ]
+    else
+      timeout = Application.get_env(:contify, :timeout, @default_timeout)
+
+      [
+        {Tesla.Middleware.BaseUrl, base_url},
+        {Tesla.Middleware.Headers, [{"user-agent", user_agent}]},
+        {Tesla.Middleware.Headers, [{"APPSECRET", app_secret}]},
+        {Tesla.Middleware.Headers, [{"APPID", app_id}]},
+        {Tesla.Middleware.Timeout, timeout: timeout},
+        {Tesla.Middleware.EncodeJson, engine: json_engine}
+        | middleware
+      ]
+    end
```

Comment:
hey @crova I think it would be best here to check if `timeout` is 0 and if so skip the Timeout Middleware.
And then put `Application.put_env(:contify, :timeout, 0)` on the `test_helpers.exs`? so that it gets configured as 0 before the test suite starts (I think)

Reply:
Thanks for the feedback @imricardoramos, I'll make the change and see how that goes.